packages/backend-dynamic-feature-service/src/manager/plugin-manager.ts

```diff
@@ -305,6 +310,69 @@ export const dynamicPluginsFeatureDiscoveryServiceFactory =
     },
   });
 
+/**
+ * A function that returns a backend feature loader that uses the dynamic plugins system to discover features.
+ *
+ * @public
+ *
+ * @example
+ * Using the `dynamicPluginsFeatureDiscoveryLoaderWithOptions` loader in a backend instance:
+ * ```ts
+ * //...
+ * import { createBackend } from '@backstage/backend-defaults';
+ * import { dynamicPluginsFeatureDiscoveryLoaderWithOptions } from '@backstage/backend-dynamic-feature-service';
+ * import { myCustomModuleLoader } from './myCustomModuleLoader';
+ *
+ * const backend = createBackend();
+ * backend.add(dynamicPluginsFeatureDiscoveryLoaderWithOptions({
+ *   moduleLoader: myCustomModuleLoader
+ * }));
+ * //...
+ * backend.start();
+ * ```
+ */
+export const dynamicPluginsFeatureDiscoveryLoaderWithOptions = (
```

Comment:
I think it's good to stick to the existing pattern of exporting these as `Object.assign(dynamicPluginsFeatureDiscoveryLoaderWithOptions, dynamicPluginsFeatureDiscoveryLoaderWithOptions())`, without exporting the `*WithOptions` version. Happy to discuss the pattern overall but let's keep things consistent.

Reply:
Done